experiments/list_unsupported.sh

```diff
@@ -34,8 +34,30 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+export ADA_INCLUDE_PATH="${ADA_INCLUDE_PATH:-/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude}"
```

Comment:
This still has a hard coded path; I don’t particularly like these, it makes it too easy for scripts to work by accident. Default paths should only be used if there is a reasonable expectation that something can actually be found at that path

Reply:
I've done something to improve this.